src/main/kotlin/ch/bda/baumannwicki/misplacedrecognizer/MisplacedTagRecognizerController.kt

```diff
@@ -0,0 +1,35 @@
+package ch.bda.baumannwicki.misplacedrecognizer
+
+import ch.bda.baumannwicki.data.LibraryCopy
+import ch.bda.baumannwicki.data.supplier.LibraryCopySupplier
+import ch.bda.baumannwicki.log.LogPersistor
+import rfid.communicationid.TagInformation
+import java.util.*
+import java.util.concurrent.atomic.AtomicBoolean
+
+class MisplacedTagRecognizerController(
+    val run: AtomicBoolean,
+    val tagInformationIncommintQueue: Queue<List<TagInformation>>,
```

Comment:
Shouldn't that be tagInformationInCommInterfaceQueue or tagInformationInCommIntQueue ?